pkg/agent/bakerapi.go

```diff
@@ -97,11 +90,23 @@ func (agentBaker *agentBakerImpl) GetLatestSigImageConfig(
 	if sigImageConfig == nil {
 		return nil, fmt.Errorf("can't find SIG image config for distro %s in region %s", distro, region)
 	}
+
+	if !distro.IsWindowsDistro() {
+		e := agentoverrides.NewEntity().WithFields(map[string]string{
+			agentoverrides.SubscriptionIDFieldName: subscriptionID,
+			agentoverrides.TenantIDFieldName:       tenantID,
+		})
+		imageVersionOverrides := agentBaker.overrides.GetLinuxNodeImageVersion(e)
+		if imageVersion, ok := imageVersionOverrides[string(distro)]; ok {
+			sigImageConfig.Version = imageVersion
```

Comment:
Is it supposed to only update the sigImageConfig.Version but not the subcriptionID?

Reply:
yep, we only override the image version here